twoliter/embedded/rpm2kit

```diff
@@ -30,3 +35,90 @@ done
 
 createrepo_c "${KIT_DIR}"
 dnf --disablerepo '*' --repofrompath "kit,file:///${KIT_DIR}" repoquery --all
+
+WORK_DIR=$(mktemp -d)
+TIMESTAMP=$(date +"%FT%T.%NZ")
+FILENAME_PREFIX="${KIT_NAME:?}--${ARCH:?}--${VERSION_ID:?}--${BUILD_ID:?}"
```

Comment:
Why the double dashes (`--`) in the filename? We don't use that convention anywhere else for output artifacts.

Reply:
typo demon strikes again